src/workers/userBanned.ts

```diff
@@ -0,0 +1,28 @@
+import { TypedWorker } from './worker';
+import { Comment, User } from '../entity';
+import { logger } from '../logger';
+import { counters } from '../telemetry';
+
+const worker: TypedWorker<'user-banned'> = {
+  subscription: 'api.user-banned',
+  handler: async ({ data }, con) => {
+    if (data.banned) {
```

Comment:
This worker can be a trigger 😉

Reply:
Trigger?

Reply:
db trigger

Reply:
Ah, handle it all on db side?

Reply:
I guess that could work as well. Was not sure if we might want to add more logic into this worker

Reply:
Your call. I don't mind either

Reply:
Refactored to use a trigger. Now it also works the other way, though I can change the trigger to only work on `true`